PowerShellTools/DebugEngine/ResourceStrings.resx

```diff
@@ -158,4 +158,10 @@
     <comment>{0} is the Progress Activity [Overall intent, eg, Removing item xyz]
 {1} is the description of the status [eg, 35 of 50 items removed]</comment>
   </data>
+  <data name="ChoicesCollectionShouldNotBeEmpty" xml:space="preserve">

```

Comment:
Nit: The PowerShell message is 

```
PS C:\WINDOWS\system32> $host.UI.PromptForChoice("test", "test2", @(), 0)
Exception calling "PromptForChoice" with "4" argument(s): ""choices" should have at least one element."
```


("choices" should have at least one element.")